packages/mathtype-ckeditor5/src/plugin.js

```diff
@@ -133,7 +134,7 @@ export default class MathType extends Plugin {
       view.bind('isEnabled').to(editor.commands.get('MathType'), 'isEnabled');
 
       view.set({
-        label: 'Insert a math equation - MathType',
+        label: StringManager.get('insert_math'),
```

Comment:
That's an excellent solution.

Also, in the case of the CKEditor5 implementation, we may use their message localization support.

Taking advantadge or the

```Javascript

    // The translation function.
    const { t } = editor.locale;

    // The localized label.
    const label = t(StringManager.get('insert_math'));


    buttonView.set( {
        label,
        ...


```

It's quite cool, since even though we could add the localization strings ourlselves, they assume the translation work themselves. 

> _In case of reusing messages, you will not need to worry about translating them as all work will be done by the CKEditor 5 team and Transifex translators. Obviously, your help in translating will still be appreciated!_

See: https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/ui/localization.html

Reply:
On a second thought, I think your approach is correct.

What I'm proposing won't work, while the solution you're proposing will work fine in all cases.

Thanks!